tests/monotouch-test/SystemConfiguration/CaptiveNetworkTest.cs

```diff
@@ -81,6 +81,8 @@ public void TryCopyCurrentNetworkInfo ()
 			// So just assert that we get null if running on iOS 12+.
 			if (TestRuntime.CheckXcodeVersion (10, 0)) {
 				Assert.IsNull (dict, "Dictionary");
+			} else if (TestRuntime.CheckXcodeVersion (9, 0)) {
+				Assert.IsNull (dict, "Dictionary"); // It doesn't seem to work in iOS 11 either, I don't know why.
```

Comment:
> An app linked against iOS 13 or later receives NULL.
https://developer.apple.com/documentation/systemconfiguration/1614126-cncopycurrentnetworkinfo

Since we're linking with the latest SDK then it's likely always null.

Reply:
I understood that differently: "apps linked against iOS 13+ receives NULL (and this is kind of implied:) _when executing on iOS 13+_".

I think the previous sentence explains it: "An app linked against iOS 12 or earlier receives a dictionary with pseudo-values." - the point is that old apps would get the new behavior if re-built with the iOS 13+ SDK (and executing on an iOS 13+ OS), not that apps build with iOS 13+ SDK will get the new behavior on existing OSes (which would require OS changes).

Reply:
Yes. It's not common but it's not the first setting I've seen that behave differently depending on which SDK version it's linked with :-)

* If you rebuild (then you re-link) then it's up to you to test your code for changes (and adapt for the null value).

* If you don't rebuild (pre-iOS 13 SDK) and run on iOS 13+ (like updating your phone) you'll get the pseudo-values so the app doesn't crash while accessing a null dictionary.

Since our test always relink then a single
```CSharp
Assert.IsNull (dict, "Dictionary");
```
is the only thing that should be required.

Reply:
@spouliot fixed